tests/test_product.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from buy_together_app import models
+# from buy_together_app import views
+
+QR = "Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+NAME = "Banana"
+DESCRIPTION = "Good!"
+
+
+@pytest.fixture
+def product0():
+    return models.Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
+
+
+class TestProductModel:
+    def test_new_product(self, product0):
```

Comment:
Are you testing the fixture here? I think this is something to avoid.

Reply:
I am testing that the fixture details are correct

Reply:
I don't see a reason to do it. Maybe it would have made sense if you had some sort of `init` method that will generate some of the members, but here you're just testing the python core functionality as far as I can tell.

Reply:
Fixed